DALI_DEPS_VERSION

```diff
@@ -1 +1 @@
-4cd59d81c91eecbea90943e34acb08dff91cb894
+b0babac0c22f8fffa6bc09936960be14601ffcb4
```

Comment:
what's this hash? As I understand the DALI_deps PR is not yet merged.

Reply:
Ah, true